### ISSUE TITLE
Still Unable to Login

### ISSUE THREAD

**Issue by Hemedy** [biezhihua/libdsm#17]
I tried this =>
To log in, translate the host address to the host name by using the inverse() method

but also returns "Unable to perform name resolution for 192.168...."

@biezhihua please help,

**Comment by biezhihua** [comment]
Check that your target host has SMB service enabled.

Also, your target host, what is the host name?

@Hemedy

**Comment by Hemedy** [comment]
@biezhihua yes SMB enabled , I tried with an app from playstore. My Hostname is "kali"

**Comment by Hemedy** [comment]
Hi @biezhihua , any luck?

**Comment by Hemedy** [comment]
Hi @biezhihua , just checking on you, If you might help soon I will appreciate.

**Comment by shingohu** [comment]
flutter login the 'host' maybe 'hostName'?
`
  Future<int> login(String host, String loginName, String password) async {
    if (_dsmId == null) {
      return 0;
    }
    int result =
        await _methodChannel.invokeMethod('DSM_login', <String, dynamic>{
      'id': _dsmId,
      'hostName': host,
      'login_name': loginName,
      'password': password,
    });
    return result;
  }
`

`
  if (call.arguments == null ||
                        call.argument<String>(ID).isNullOrEmpty() ||
                        call.argument<String>(HOST_NAME).isNullOrEmpty() ||
                        call.argument<String>(LOGIN_NAME).isNullOrEmpty() ||
                        call.argument<String>(PASSWORD).isNullOrEmpty()
                ) {
                    result.error("PARAM_ERROR", "Illegal parameter", null)
                }

`
the native code HOST_NAME is 'hostName'
